package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/skyux-lib-help",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Because we changed some of the exports/declarations, to be safe we may want to make this a Major version change, has this been tested in builder?  Should we make this a beta release first before going directly to major version?

Reply:
This has not been tested in builder, so I think a beta build would make sense.

Reply:
👍